misk/src/main/kotlin/misk/security/authz/fake/FakeCallerAuthenticator.kt

```diff
@@ -12,14 +12,18 @@ import javax.inject.Singleton
 @Singleton
 class FakeCallerAuthenticator : MiskCallerAuthenticator {
   @Inject lateinit var currentRequest: ActionScoped<misk.web.Request>
+  @Inject lateinit var fakeAnnotations: List<FakeAccess>
 
   override fun getAuthenticatedCaller(): MiskCaller? {
```

Comment:
... if there are no headers set, return the fake MiskCaller

Reply:
done!